### PR TITLE
Updates from Julien

### PR DESCRIPTION
**Pull request by cescoffier**

### CODE REVIEW

File: audit-service/src/main/java/io/vertx/workshop/audit/impl/AuditVerticle.java [hunk 1]
```diff
@@ -87,10 +87,7 @@ private void retrieveOperations(RoutingContext context) {
 
   private Future<MessageConsumer<JsonObject>> retrieveThePortfolioMessageSource() {
     Future<MessageConsumer<JsonObject>> future = Future.future();
-    MessageSource.getConsumer(discovery,
-        new JsonObject().put("name", "portfolio-events"),
-        future.completer()
-    );
+    MessageSource.getConsumer(discovery, new JsonObject().put("name", "portfolio-events"), future);
```
Comment (cescoffier) [inline audit-service/src/main/java/io/vertx/workshop/audit/impl/AuditVerticle.java]: should we switch the audit to RX completely ?
We can use rxGetConsumer

Comment (vietj) [inline audit-service/src/main/java/io/vertx/workshop/audit/impl/AuditVerticle.java]: yes I will make another pass tonight

**Comment by vietj** [comment]
I need to rollback the commit for RxJava and Portfolio and then finish the Audit with Rx.

Then I can start looking at the doc

**Comment by vietj** [comment]
I've reverted the portfolio rx commit

File: audit-service/src/main/java/io/vertx/workshop/audit/impl/AuditVerticle.java (hunk 1 above)
Comment (vietj) [inline audit-service/src/main/java/io/vertx/workshop/audit/impl/AuditVerticle.java]: so actually I have only rx-ified the solution and not yet copied to the audit verticle, will

Comment (vietj) [inline audit-service/src/main/java/io/vertx/workshop/audit/impl/AuditVerticle.java]: copy it

**Pull request merged**